crates/turbopack-core/src/issue/mod.rs

```diff
@@ -410,9 +410,22 @@ pub struct LazyIssueSource {
 
 #[turbo_tasks::value_impl]
 impl LazyIssueSource {
+    /// Create a [`LazyIssueSource`] from byte offsets given by an swc ast node
+    /// span.
+    ///
+    /// Arguments:
+    ///
+    /// * `source`: The source code in which to look up the byte offsets.
+    /// * `start`: The start index of the span. Must use **1-based** indexing.
+    /// * `end`: The end index of the span. Must use **1-based** indexing.
     #[turbo_tasks::function]
-    pub fn new(source: Vc<Box<dyn Source>>, start: usize, end: usize) -> Vc<Self> {
-        Self::cell(LazyIssueSource { source, start, end })
+    pub fn from_swc_offsets(source: Vc<Box<dyn Source>>, start: usize, end: usize) -> Vc<Self> {
+        println!("received {} {}", start, end);
+        Self::cell(LazyIssueSource {
+            source,
+            start: start - 1,
```

Comment:
Could we somehow handle the case when start or end are 0? Currently this will panic in that case in dev, or gives weird numbers in release